package.json

```diff
@@ -1,29 +1,29 @@
 {
-  "name": "bitmovin-player-analytics-comscore",
+  "name": "@bitmovin/player-integration-comscore",
```

Comment:
I don't like the name. What if comscore would have a second product besides analytics? Then the names would intuitively be:

 * `bitmovin-player-analytics-comscore`
 * `bitmovin-player-somethingelse-comscore`

What would it be with the current pattern? `player-integration-comscore-somethingelse`? Then the current package should be named `player-integration-comscore-analytics`. But since the naming idea was initially to form a tree, and "comscore analytics" os a specific "analytics" product, the order should be the other way: `bitmovin (root) -> player (product) -> kind -> company`.

Reply:
https://github.com/bitmovin/player-issues-web/issues/1858#issuecomment-471993156